front/src/Components/EventBox.js

```diff
@@ -50,33 +190,45 @@ class EventBox extends React.Component {
               <ListItemText primary={`${event.accepted}人/定員${event.limit}人`} />
             </ListItem>
             <ListItem>
-              <Grid container direction="row" justify="flex-star" align="center">
-                {
-                  event.tags.map( item => (
-                    <Grid item>
-                      <Button dense className={classes.button}>
-                        {item}
-                      </Button>
-                    </Grid>
-                  ))
-                }
+              <Grid container direction="row" justify="flex-start" align="center">
+                {event.tags.map(item =>
+                  <Grid item>
+                    <Button dense className={classes.button}>
+                      {item}
+                    </Button>
+                  </Grid>,
+                )}
               </Grid>
             </ListItem>
             <Divider />
             <ListItem>
               <Grid container direction="row" justify="space-between" align="center">
                 <Grid item>
-                  <Button dense className={classes.button} onClick={() => { handleEdit(event); }}>
-                    詳細を見る
+                  <Button
+                    dense
+                    className={classes.button}
+                    onClick={() => {
+                      this.handleOpenDrawer();
+                    }}
+                  >
+                    {this.state.isOpenDrawer ? '閉じる' : '詳細を見る'}
                   </Button>
                 </Grid>
                 <Grid item>
-                  <Button dense className={classes.button}>
-                    Compass
-                  </Button>
+                  <ListItemSecondaryAction>
+                    <IconButton
+                      style={{ width: 100 }}
+                      onClick={() => {
+                        handleProviderJump(event);
+                      }}
+                    >
+                      <img style={{ width: 100 }} src={`${logoUrl}`} />
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text